### PR TITLE
Landingpage

### PR DESCRIPTION
**Pull request by brettgaylor**
Adding links to current starter projects, and buttons for "start from scratch" and "tutorial"

### CODE REVIEW

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @k88hudson received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/b12fcd9ca19fea6/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Success

Full output at http://botio.butyr.org:8000/b12fcd9ca19fea6/output.txt

Total script time: 0.85 mins

Server is running on http://pr1103.butyr.org/templates/basic/
Run the Test Suite: http://pr1103.butyr.org/test/

File: public/media/css/sandstone.css [hunk 1]
```diff
@@ -107,6 +107,79 @@ body {
   color: #333333;
   background: #fff;
 }
+
+.button, .button:link, .button:visited {
+    -moz-transition: all 0.25s linear 0s;
```
Comment (k88hudson) [inline public/media/css/sandstone.css]: If you're doing `-moz-transition`, do:

```
-webkit-transition
-moz-transition
-o-transition
-ms-transition
transition
```

File: public/media/css/sandstone.css [hunk 2]
```diff
@@ -107,6 +107,79 @@ body {
...
+    background-color: #659324;
+    background-image: -moz-linear-gradient(#81BC2E, #659324);
+    background-repeat: repeat-x;
+    border: 0 none;
+    border-radius: 0.25em 0.25em 0.25em 0.25em;
+    box-shadow: 0 2px 0 0 rgba(0, 0, 0, 0.1), 0 -2px 0 0 rgba(0, 0, 0, 0.2) inset;
+    color: #FFFFFF;
+    display: inline-block;
+    font-family: 'OpenSansRegular',sans-serif;
```
Comment (k88hudson) [inline public/media/css/sandstone.css:121]: Remove

File: public/media/css/sandstone.css [hunk 3]
```diff
@@ -107,6 +107,79 @@ body {
   color: #333333;
   background: #fff;
 }
+
+.button, .button:link, .button:visited {
+    -moz-transition: all 0.25s linear 0s;
+    background-color: #659324;
+    background-image: -moz-linear-gradient(#81BC2E, #659324);
```
Comment (k88hudson) [inline public/media/css/sandstone.css:114]: Remove this if you're not doing all cross-browser gradients

File: public/media/css/sandstone.css [hunk 4]
```diff
@@ -107,6 +107,79 @@ body {
...
+    font-size: 14px;
+    height: 48px;
+    line-height: 48px;
+    padding: 0 24px;
+    text-align: center;
+    text-decoration: none;
+    text-shadow: 0 1px 0 rgba(0, 0, 0, 0.25);
+}
+.button-blue:hover, .button-blue:link:hover, .button-blue:visited:hover {
+    -moz-transition: all 0.25s linear 0s;
```
Comment (k88hudson) [inline public/media/css/sandstone.css]: same thing here

File: public/media/css/sandstone.css [hunk 5]
```diff
@@ -107,6 +107,79 @@ body {
...
+    box-shadow: 0 2px 0 0 rgba(0, 0, 0, 0.1), 0 -2px 0 0 rgba(0, 0, 0, 0.2) inset, 0 12px 24px 2px #83C822 inset;
+    color: #FFFFFF;
+    text-decoration: none;
+}
+.button-blue:active, .button-blue:link:active, .button-blue:visited:active {
+    -moz-transition: all 0.25s linear 0s;
```
Comment (k88hudson) [inline public/media/css/sandstone.css]: same thing here

File: public/media/css/sandstone.css [hunk 6]
```diff
@@ -107,6 +107,79 @@ body {
...
+    background-color: #276195;
+    background-image: -moz-linear-gradient(#3C88CC, #276195);
+    background-repeat: repeat-x;
+    border: 0 none;
+    border-radius: 0.25em 0.25em 0.25em 0.25em;
+    box-shadow: 0 2px 0 0 rgba(0, 0, 0, 0.1), 0 -2px 0 0 rgba(0, 0, 0, 0.2) inset;
+    color: #FFFFFF;
+    display: inline-block;
+    font-family: 'OpenSansRegular',sans-serif;
```
Comment (k88hudson) [inline public/media/css/sandstone.css:152]: Remove

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_check** from @humphd received. Current queue size: 0

Live output at: http://botio.butyr.org:8000/a7583171edcd407/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Received

Command **cmd_preview** from @humphd received. Current queue size: 1

Live output at: http://botio.butyr.org:8000/e90c487100aa8da/output.txt

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/a7583171edcd407/output.txt

Total script time: 3.00 mins

**Comment by ghost** [comment]
#### From: Bot.io (butyr.org)

---

# Failed

Full output at http://botio.butyr.org:8000/e90c487100aa8da/output.txt

Total script time: 3.00 mins

File: public/index.html [hunk 7]
```diff
@@ -42,25 +42,27 @@
...
                 or sounds on SoundCloud, add your own comments and links, or drag and drop in
                 content from all over the web—right in your web browser.
               </p>
               <p>
-                The result is a whole new way of telling stories on the web—videos that are dynamic,
+                The result is a whole new way of telling stories on the web — videos that are dynamic,
                 full of links, and unique each time you watch them. It's video beyond the box.
               </p>
+              <p class="intro-bar">
+				<a class="button-blue start" href="/templates/basic">Start from scratch</a>
```
Comment (humphd) [inline public/index.html]: some indenting issues in these lines, should be moved over to the right.

**Pull request closed without merging**